### PR TITLE
Update AWS example

### PR DESCRIPTION
**Pull request by cdmadrigal**
Update AWS LB example with README to better match other examples provided.

### CODE REVIEW

**Review by rvelaVenafi — APPROVED** [review]

**Review by luispresuelVenafi — APPROVED** [review]

**Comment by luispresuelVenafi** [comment]
Hi @cdmadrigal , we approved but we cannot merge as we have enabled requirements in Github to do commit signing before merging PR's

**Pull request merged**
